pkg/cmd/pulumi/preview.go

```diff
@@ -138,7 +138,7 @@ func newPreviewCmd() *cobra.Command {
 				return result.FromError(err)
 			}
 
-			m, err := getUpdateMetadata(message, root, execKind, execAgent)
+			m, err := getUpdateMetadata(message, root, execKind, execAgent, "")
```

Comment:
Should we send something here if we're using preview to generate a plan? "update_plan": "generating"?

Reply:
I think we could use the same field `updatePlane: true` if we are generating an update plan as a part of the preview.